app/views/root/index.html.erb

```diff
@@ -162,10 +162,10 @@
             </div>
             <div class="promo-image">
               <div class="promo-content">
-                <a href="https://www.blog.gov.uk"><%= image_tag 'homepage/govuk-blogs.jpg', alt: 'GOV.UK blogs' %></a>
-                <h3>GOV.UK blogs</h3>
+                <a href="/taxfreesavings"><%= image_tag 'homepage/tax-free-savings.png', alt: 'Tax-free savings' %></a>
+                <h3>Tax-free savings</h3>
                 <p>
-                  <a href="https://www.blog.gov.uk">Search the list</a> of GOV.UK blogs, and find one to match your interest.
+                  More people will be able to get tax-free interest on their savings from 6 April - <a href="/taxfreesavings">find out if you'll qualify</a>.
```

Comment:
Yup - amended. And there was me getting pedantic about the hyphen in the text to insert instead of a proper dash...